src/utils/eyeToggle.js

```diff
@@ -0,0 +1,44 @@
+import { IconButton, InputAdornment } from '@material-ui/core';
+import { Visibility, VisibilityOff } from '@material-ui/icons';
+import React from 'react';
+
+export function toggleInputType(e, showPass) {
+  const input = e.currentTarget.parentElement.previousSibling;
+  if (input.type === 'password') {
+    input.type = 'text';
+    showPass(false);
+  } else {
+    input.type = 'password';
+    showPass(true);
+  }
+}
+
+export function endAdornment(
+  isVisible,
+  setShowPass,
+  inputTextColor,
+  outlineClass
+) {
+  return {
+    endAdornment: (
+      <InputAdornment position='end'>
+        <IconButton
+          aria-label='toggle password visibility'
+          onClick={(e) => toggleInputType(e, setShowPass)}
```

Comment:
make it onClick={eyeHendler}

Reply:
It should recieve 2 args, not 1, that is event.